packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -34,6 +35,7 @@ import {
     getWrappedComponentsListener,
     getComponentAsString,
     getTemplateReactiveObserver,
+    ComponentConstructor,
```

Comment:
can the 'Component' part be removed? -> import { Interface, Constructor } from 'component'

Reply:
this part is being refactored as part of #1386 , we shoudl probably wait for that one to land.